debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/SourceInfo.java

```diff
@@ -21,8 +21,8 @@
     public static final String LCR_POSITION_KEY = "lcr_position";
     public static final String SNAPSHOT_KEY = "snapshot";
 
-    private long scn;
-    private Long commitScn;
+    private String scn;
+    private String commitScn;
```

Comment:
I'd still suggest to use `Scn` here, and only convert it to String in `OracleSourceInfoStructMaker.struct()` (similar to the conversion of `Instant` in `commonStruct()`.